### ISSUE TITLE
Integration tests break mvn verify (without additional, undocumented setup)

### ISSUE THREAD

**Issue by elharo** [googleapis/java-bigquery#28]
With the inclusion of the samples into the repo, a simple clone and `mvn verify` no longer works. Undoubtedly there's some extra setup required. Ideally there shouldn;t be, but if this is mandatory then the needed steps need to be clearly documented in the relevant files in the repo.

```
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.002 s <<< FAILURE! - in com.google.cloud.bigquery.it.ITBigQueryTest
[ERROR] com.google.cloud.bigquery.it.ITBigQueryTest  Time elapsed: 1.001 s  <<< ERROR!
com.google.cloud.storage.StorageException: 401 Unauthorized
	at com.google.cloud.bigquery.it.ITBigQueryTest.beforeClass(ITBigQueryTest.java:290)
Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 401 Unauthorized
	at com.google.cloud.bigquery.it.ITBigQueryTest.beforeClass(ITBigQueryTest.java:290)

[ERROR] com.google.cloud.bigquery.it.ITBigQueryTest  Time elapsed: 1.002 s  <<< ERROR!
com.google.cloud.bigquery.BigQueryException: Request is missing required authentication credential. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
	at com.google.cloud.bigquery.it.ITBigQueryTest.afterClass(ITBigQueryTest.java:323)
Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 
401 Unauthorized
{
  "code" : 401,
  "errors" : [ {
    "domain" : "global",
    "location" : "Authorization",
    "locationType" : "header",
    "message" : "Login Required.",
    "reason" : "required"
  } ],
  "message" : "Request is missing required authentication credential. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.",
  "status" : "UNAUTHENTICATED"
}
	at com.google.cloud.bigquery.it.ITBigQueryTest.afterClass(ITBigQueryTest.java:323)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR] com.google.cloud.bigquery.it.ITBigQueryTest.com.google.cloud.bigquery.it.ITBigQueryTest
[ERROR]   Run 1: ITBigQueryTest.beforeClass:290 » Storage 401 Unauthorized
[ERROR]   Run 2: ITBigQueryTest.afterClass:323 » BigQuery Request is missing required authentic...
[INFO] 
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
[INFO] 
```

**Comment by chingor13** [comment]
As discussed in our sync up, these ones are just the normal integration test suite. We will be changing the integration test execution to be opt-in rather than opt-out.

**Comment by BenWhitehead** [comment]
A fix for this is in https://github.com/googleapis/java-shared-config/pull/75

**Comment by stephaniewang526** [comment]
https://github.com/googleapis/java-bigquery/commit/29ede0ff107f9e39da38953ad2e785b67722b769 this should have fixed this issue @elharo

**Comment by BenWhitehead** [comment]
Not yet fully fixed, still need the bump on shared-config to go out.

**Comment by BenWhitehead** [comment]
Released in shared-config [`v0.4.0`](https://github.com/googleapis/java-shared-config/releases/tag/v0.4.0)
